src/shared/regions/defaultRegionProvider.ts

```diff
@@ -4,98 +4,59 @@
  */
 
 import * as vscode from 'vscode'
-import { endpointsFileUrl } from '../constants'
-import { ext } from '../extensionGlobals'
-import { getLogger, Logger } from '../logger'
-import { CompositeResourceFetcher } from '../resourcefetcher/compositeResourceFetcher'
-import { FileResourceFetcher } from '../resourcefetcher/fileResourceFetcher'
-import { HttpResourceFetcher } from '../resourcefetcher/httpResourceFetcher'
-import { ResourceFetcher } from '../resourcefetcher/resourcefetcher'
+import { Endpoints, Region } from './endpoints'
+import { EndpointsProvider } from './endpointsProvider'
 import { RegionInfo } from './regionInfo'
 import { RegionProvider } from './regionProvider'
 
-export interface RawRegion {
-    description: string
-}
+export class DefaultRegionProvider implements RegionProvider {
+    private readonly onRegionProviderUpdatedEmitter: vscode.EventEmitter<void> = new vscode.EventEmitter()
+    private _loadedRegions: RegionInfo[] = []
```

Comment:
nit: `_` before name, please no

Reply:
Agreed, older code has _ prefixes. This code appears as new in the delta but is actually existing code. My next PR will contain name sanitizing in this class.